sematic/config/user_settings.py

```diff
@@ -25,13 +24,19 @@ class UserSettingsVar(AbstractSettingsVar):
     AWS_S3_BUCKET = "AWS_S3_BUCKET"
 
 
+CLI_COMMAND = "settings"
```

Comment:
This constant is only used once, and the server settings don't have an equivalent, as they specify the string directly. Please standardize.

Reply:
It is used outside of this module too.